File: 1700.cpp
#include<bits/stdc++.h>

using namespace std;


int countStudents(vector<int> & students, vector<int>& sandwiches){
  unordered_map<int,int> mpp;
  for(auto it: students){
    mpp[it]++;
  }
  int res = sandwiches.size();
  for(auto elem: sandwiches){
    if(mpp.find(elem)!=mpp.end() && mpp[elem]>0){
      mpp[elem]-=1;
      res--;
    }else{
      return res;
    }

  }
  return res;
}


int main()
{
  vector<int> m({1,1});
  vector<int> n({0,1});
  int res = countStudents(m,n);
  cout<<res;
  return 0;
}

File: 1700.py
from collections import Counter
def countStudents(students,sandwiches):
    dicti= Counter(students)
    res = len(sandwiches)
    for elem in sandwiches:
        if elem in dicti.keys():
            if(students[elem]>0):
                dicti[elem]-=1
                res-=1
            else:
                return res
        else:
            return res
    return res


m=[1,1]
n=[0,1]
print(countStudents(m,n))



